packages/gatsby/cache-dir/static-entry.js

```diff
@@ -334,11 +326,7 @@ export default (pagePath, callback) => {
     })
 
   // Add page metadata for the current page
-  const windowData = `/*<![CDATA[*/window.page=${JSON.stringify(page)};${
-    page.jsonName in dataPaths
-      ? `window.dataPath="${dataPaths[page.jsonName]}";`
-      : ``
-  }/*]]>*/`
+  const windowData = `/*<![CDATA[*/window.pageData=${pageDataRaw};/*]]>*/`
```

Comment:
We shouldn't be putting the entire `pageDataRaw` into the window data as this includes the query results, which could be quite large

Reply:
Addressed in 7f5605af5